encode.py

```diff
@@ -14,11 +14,15 @@ def __init__(self,
         self.default_traversal_path = default_traversal_path
 
     @requests
-    def encode(self, docs: DocumentArray, **kwargs):
-        if docs:
+    def encode(self, docs: DocumentArray, parameters: dict, **kwargs):
+        assert(len(docs)==1)
+        traversal_path = parameters.get('traversal_path', self.default_traversal_path)
+        flattened_docs = docs.traverse_flat(traversal_path)
```

Comment:
traversal paths should be plurarl

Reply:
but here I am using only one traversal path

Reply:
ok, I will generalize it to plural